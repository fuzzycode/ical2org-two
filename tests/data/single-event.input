BEGIN:VCALENDAR
PRODID:-//Google Inc//Google Calendar 70.9054//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:PUBLISH
X-WR-CALNAME:Tester
X-WR-TIMEZONE:Europe/Paris
X-WR-CALDESC:Test how to export to org mode
BEGIN:VTIMEZONE
TZID:Europe/Paris
X-LIC-LOCATION:Europe/Paris
BEGIN:DAYLIGHT
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
DTSTART:19700329T020000
RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
END:DAYLIGHT
BEGIN:STANDARD
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
DTSTART:19701025T030000
RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
END:STANDARD
END:VTIMEZONE
BEGIN:VEVENT
DTSTART:20190212T180000Z
DTEND:20190212T190000Z
DTSTAMP:20190216T084259Z
UID:2l0654150e5qsu5r28se258np4
CREATED:20190211T175520Z
DESCRIPTION:
LAST-MODIFIED:20190211T175520Z
LOCATION: Some place
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Some description
TRANSP:OPAQUE
END:VEVENT
END:VCALENDAR
