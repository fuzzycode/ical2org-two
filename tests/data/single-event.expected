# -*- buffer-read-only: t -*-
#+PROPERTY: PRODID -//Google Inc//Google Calendar 70.9054//EN
#+PROPERTY: VERSION 2.0
#+PROPERTY: CREATED [2019-02-05 Tue 22:03]

* Test Event One Time Limited
%%(and (diary-block 02 12 2019 02 12 2019)) 19:00--20:00 Test Event One Time Limited
        :PROPERTIES:
        :ID: 2l0654150e5qsu5r28se258np4
        :LOCATION:
        :CREATED: [2019-02-11 Mon 18:55]
        :LAST-MODIFIED: [2019-02-11 Mon 18:55]
        :END:
